aiida/backends/general/migrations/duplicate_uuids.py

```diff
@@ -44,17 +47,21 @@ def verify_uuid_uniqueness(table):
         )
 
 
-def apply_new_uuid_mapping(table, mapping):
+def _apply_new_uuid_mapping(table, mapping):
     """Take a mapping of pks to UUIDs and apply it to the given table.
 
     :param table: database table with uuid column, e.g. 'db_dbnode'
     :param mapping: dictionary of UUIDs mapped onto a pk
     """
+    from aiida.manage.manager import get_manager
     backend = get_manager().get_backend()
-    backend.query_manager.apply_new_uuid_mapping(table, mapping)
+    for pk, uuid in mapping.items():
+        query = f"""UPDATE {table} SET uuid = '{uuid}' WHERE id = {pk}"""
+        with backend.cursor() as cursor:
+            cursor.execute(query)
 
 
-def deduplicate_uuids(table=None, dry_run=True):
+def deduplicate_uuids(table=None, dry_run=False):
```

Comment:
I wouldn't change this default. Is there a particular reason you did so?

Reply:
Ok done, it doesn't make too much difference because this function is only used two places in the code `aiida/cmdline/commands/cmd_database.py` (where dry_run is set explicitly) and `tests/backends/aiida_django/migrations/test_migrations_many.py` (where it was assuming `dry_run=False`, but now I've set it explicitly 962175d)